lib/endpoints/class-wp-rest-controller.php

```diff
@@ -127,10 +127,12 @@ public function delete_item( $request ) {
 	/**
 	 * Prepare the item for create or update operation.
 	 *
-	 * @param WP_REST_Request $request Request object.
-	 * @return WP_Error|object $prepared_item
+	 * @param WP_REST_Request $request  Request object.
+	 * @param bool            $creating Whether the item is being created.
```

Comment:
> wp_insert_post uses $updating instead of $creating:

This is unrelated to this PR in some ways but every hook in the api is using $creating should we change it to $updating and reverse the booleans to follow with the rest of wp?